dashboard/app/views/home/_tos.md

```diff
@@ -13,7 +13,7 @@ These Terms are subject to change. You are agreeing to be bound by such changes
 Some Services may be subject to additional posted guidelines, rules or terms of service (“**Additional Terms**”) and your use of such Services will be conditioned on your agreement to the Additional Terms. If there is any conflict between these Terms and the Additional Terms, the Additional Terms will control for that Service, unless the Additional Terms expressly state that these Terms will control.
 
 ### Children Under Age 13
-We take steps to minimize the collection of personal data from Users under the age of 13 as explained in the Code.org [privacy policy](/upcomingPrivacy). If you are under the age of 13, you will be able to use our App Lab tool and our upcoming Game Lab tool in the classroom with your teacher’s approval.  Users under the age of 13 will not be permitted to use these tools for outside the classroom use. Users under 13 can try most tutorials without creating an account, but your learning progress won’t be saved. If you are under the age of 13, we encourage you to get the approval of a parent or legal guardian before creating an account on Code.org.  Code.org allows certain representatives of school districts and schools, such as teachers, administrators, counselors, and other educators (a “**Educator**”) to connect with students through the Services in order to provide tutorial, educational and similar services. If you are an Educator, you may register accounts on the Services for one or more of your students.
+We take steps to minimize the collection of personal data from Users under the age of 13 as explained in the Code.org [privacy policy](/privacy). If you are under the age of 13, you will be able to use our App Lab tool and our upcoming Game Lab tool in the classroom with your teacher’s approval.  Users under the age of 13 will not be permitted to use these tools for outside the classroom use. Users under 13 can try most tutorials without creating an account, but your learning progress won’t be saved. If you are under the age of 13, we encourage you to get the approval of a parent or legal guardian before creating an account on Code.org.  Code.org allows certain representatives of school districts and schools, such as teachers, administrators, counselors, and other educators (an “**Educator**”) to connect with students through the Services in order to provide tutorial, educational and similar services. If you are an Educator, you may register accounts on the Services for one or more of your students.
```

Comment:
Actually since this is in dashboard, it'll need the code_org_url function because right now it's a link relative to studio.code.org and studio.code.org/privacy doesn't exist.

`[privacy policy](<%= CDO.code_org_url('/privacy') %>)`